src/main/java/com/marafone/marafone/game/active/ActiveGameController.java

```diff
@@ -32,11 +33,23 @@ public List<Game> getPublicGames(){
         return activeGameService.getPublicGames();
```

Comment:
We don't need getPublicGames when you add getWaitingGames, so you can remove it.